util/io.go

```diff
@@ -45,3 +45,21 @@ func UniqueDirectory(path string) (uniquePath string, err error) {
 		return true, nil
 	})
 }
+
+// CreateFileIfNotExists creates a file if it does not already exists.
+func CreateFileIfNotExists(filePath string) (file *os.File, err error) {
```

Comment:
I think https://pkg.go.dev/os#OpenFile does that already... now that I think about it.